src/services/administration/iam/role/modules/role-management-tab/RoleDetail.vue

```diff
@@ -0,0 +1,158 @@
+            id: MENU_ID.ASSET_INVENTORY,
+            title: MENU_INFO_MAP[MENU_ID.ASSET_INVENTORY].label,
+            fields: computed(() => [
+                { name: MENU_ID.ASSET_INVENTORY_CLOUD_SERVICE, label: MENU_INFO_MAP[MENU_ID.ASSET_INVENTORY_CLOUD_SERVICE].label },
+                { name: MENU_ID.ASSET_INVENTORY_COLLECTOR, label: MENU_INFO_MAP[MENU_ID.ASSET_INVENTORY_COLLECTOR].label },
+                { name: MENU_ID.ASSET_INVENTORY_SERVICE_ACCOUNT, label: MENU_INFO_MAP[MENU_ID.ASSET_INVENTORY_SERVICE_ACCOUNT].label },
+            ]),
+            data: {} as any,
+        });
+        const costExplorerState = reactive<DefinitionTableState>({
+            id: MENU_ID.COST_EXPLORER,
+            title: MENU_INFO_MAP[MENU_ID.COST_EXPLORER].label,
+            fields: computed(() => [
+                { name: MENU_ID.COST_EXPLORER_DASHBOARD, label: MENU_INFO_MAP[MENU_ID.COST_EXPLORER_DASHBOARD].label },
+                { name: MENU_ID.COST_EXPLORER_COST_ANALYSIS, label: MENU_INFO_MAP[MENU_ID.COST_EXPLORER_COST_ANALYSIS].label },
+                { name: MENU_ID.COST_EXPLORER_BUDGET, label: MENU_INFO_MAP[MENU_ID.COST_EXPLORER_BUDGET].label },
+            ]),
+            data: {} as any,
+        });
+        const alertManagerState = reactive<DefinitionTableState>({
+            id: MENU_ID.ALERT_MANAGER,
+            title: MENU_INFO_MAP[MENU_ID.ALERT_MANAGER].label,
+            fields: computed(() => [
+                { name: MENU_ID.ALERT_MANAGER_DASHBOARD, label: MENU_INFO_MAP[MENU_ID.ALERT_MANAGER_DASHBOARD].label },
+                { name: MENU_ID.ALERT_MANAGER_ALERT, label: MENU_INFO_MAP[MENU_ID.ALERT_MANAGER_ALERT].label },
+                { name: MENU_ID.ALERT_MANAGER_ESCALATION_POLICY, label: MENU_INFO_MAP[MENU_ID.ALERT_MANAGER_ESCALATION_POLICY].label },
+            ]),
+            data: {} as any,
+        });
+        const administrationState = reactive<DefinitionTableState>({
+            id: MENU_ID.ADMINISTRATION,
+            title: MENU_INFO_MAP[MENU_ID.ADMINISTRATION].label,
+            fields: computed(() => [
+                { name: MENU_ID.ADMINISTRATION_USER, label: `IAM > ${MENU_INFO_MAP[MENU_ID.ADMINISTRATION_USER].label}` },
+                { name: MENU_ID.ADMINISTRATION_ROLE, label: `IAM > ${MENU_INFO_MAP[MENU_ID.ADMINISTRATION_ROLE].label}` },
+                { name: MENU_ID.ADMINISTRATION_POLICY, label: `IAM > ${MENU_INFO_MAP[MENU_ID.ADMINISTRATION_POLICY].label}` },
+            ]),
+            data: {} as any,
+        });
+        const getRoleDetailData = async (roleId) => {
+            baseInfoState.loading = true;
+            try {
+                const res = await SpaceConnector.client.identity.role.get({
+                    role_id: roleId,
+                });
+                baseInfoState.data = res;
+                baseInfoState.loading = false;
+            } catch (e) {
+                ErrorHandler.handleError(e);
+            }
+        };
```

Comment:
```suggestion
        const getRoleDetailData = async (roleId) => {
            baseInfoState.loading = true;
            try {
                const res = await SpaceConnector.client.identity.role.get({
                    role_id: roleId,
                });
                baseInfoState.data = res;
            } catch (e) {
                ErrorHandler.handleError(e);
                baseInfoState.data = {};
            } finally {
                baseInfoState.loading = false;
            }
        };
```

중간에 에러가 나서 catch 문을 탈 경우 loading이 계속 보여질 수 있으므로, finally 블록에 넣어주어야 합니다.